Chapters/2-4-Chapter11.md

```diff
@@ -0,0 +1,169 @@
+
+Hae doubtful eyes peered deep into his mind, tore down his defenses. Thi could tell that he wasn't entirely invested in all of this.
+
+"Linyu!" Casra rebuked hae, but Kipnu wasn't offended. He knew there was a shred of truth in hae words; a truth that he couldn't stop feeling guilty about.
+
+"I could never forget about you, Linyu. Or your mother, or your nather. When I find a way to get my family back, I'll bring them here to tell you the stories."
+
+"That's what every no-kano says. They never come back."
+
+Kipnu felt a small tear welling up. He wanted to say _I'm not one of those people. I mean the things that I say_. But instead he cracked a small smile to cover it up.
+
+"You know what I do when I miss my family?" he asked and Linyu shook hae head. He reached behind his neck and lifted a pendant from his neck, cherishing the nicks and scratches it had gathered over the years.
+
+"I hold onto this, and everything is okay again."
+
+"What is it?" thi asked.
+
+"It's a binding pendant. It represents the bond I share with a maman and a woman."
+
+"Do they have them too?" thi asked.
+
+"Of course," he said. "So they can feel comforted when they miss me."
+
+He carefully, put it around hae neck, and thi turned it over in hae fingers, looking at the detail in the engravings.
+
+He told hae, "If you want, you can borrow it for a while. So you too have something to hold onto."
+
+Thi smiled with a glance and said, "Thanks!", looking back down to observe all its odds and ends. Casra was smiling as thi came closer.
+
+Thi said, "Okay Linyu, head back in or you'll freeze out here."
+
+Without taking hae focus off the pendant, Linyu walked toward the hole and descended back into the house, leaving Casra and Kipnu to speak.
+
+"Thank you for that," thi said. "You really didn't have to."
+
+"It's no big deal. I'd like to be a father someday. Call it practice."
+
+"You'd make a great one," thi said.
+
+An easy pause was held between them as they listened to the quiet in the air.
+
+"Casra," he asked. "If I told you the world was ending. Would you still have come back?"
+
+Hae eyes perked up, understanding the message he was trying to send.
+
+"If it meant Setre and the others had even one day of freedom, it would still be worth it."
+
+Kipnu nodded in understanding.
+
+But thi added, "I do wonder if bringing Linyu was a bad idea. Thi's old enough to understand the end of the world, but not quite ready to handle hae own mortality."
+
+He responded politely, "If you don't mind me saying, your kaln is much more mature than any of hae age I've ever met."
+
+Casra smiled, laughed inside haeself. "Linyu is still my little baby, I guess." But hae troubled head took hold again.
+
+"How will it happen?" thi asked.
+
+Kipnu hesitated. "Ara. Something they call the obscura."
+
+"I see..." thi said quizzically.
+
+Thi arose to head back inside, but before thi stepped onto the ladder, thi called to him, "I can get you back home tonight on one condition."
+
```

Comment:
So the previous chapter where Casra said there's a low chance of going back was a lie?